main.go

```diff
@@ -175,13 +208,19 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-
 	// create the faucet instance
-	f := faucet{
-		signer:     &signer,
-		authTypes:  authTypes,
-		waitPeriod: waitPeriod,
-		storage:    storage,
+	f := faucet.Faucet{
+		Signer:     &signer,
+		AuthTypes:  authTypes,
+		WaitPeriod: waitPeriod,
+		Storage:    storage,
+	}
+	var s *stripehandler.StripeHandler
+	if stripeKey != "" && stripeWebhookSecret != "" {
```

Comment:
The check to enable stripe, must be if enabled on the authTypes, not if the vars are set.